releasenotes/notes/add-enqueue_many-support-to-rq-contrib-1e20380519c153d4.yaml

```
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    tracing: adds support for `enqueue_many` in `rq` versions 1.14.0 and above. The underlying job queuing function had
```

I think we want double tick marks for release notes.

```suggestion
    tracing: adds support for ``enqueue_many`` in ``rq`` versions 1.14.0 and above. The underlying job queuing function had
```